fixing pages layout style

blackd44 opened the pull request:
# PR Description
fixing the layout of `header`, `side navigation`, `main content` and `footer`

# Description of tasks that were expected to be completed
- Removing side navigation on the top of other contents

# How has this been tested?
**for local**
- clone the github
- run `npm install`
- run `npm run dev`
- open your browser to open the local Website

**for deployed version**
- open the link below: https://metron-devpulse-git-fix-pages-layout-metron.vercel.app/

**after opening the browser
You must get on each page and check if page layout, and each element is placed well in the browser.

# Number of Commits
**1 commits**

# Screenshots (If appropriate)
- page layout
   <img width="1440" alt="Screenshot 2023-08-31 at 08 39 19" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/112763782/2beb6eb7-2a9e-40fd-a3dd-8ebc9736845d">
- models
   <img width="1440" alt="Screenshot 2023-08-31 at 08 47 10" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/112763782/a39b15fa-4477-4886-a100-47f82739870c">


# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

ceelogre commented:
The table is still not centered.
![image](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/5022823/6afd0461-334d-4464-9fa6-19c85696d0f1)

edwinnambaje reviewed (approved):
LGTM

AbdulKhaliq59 reviewed (approved):
LGTM

blackd44 committed:
fix(layout style):
- adding pages layout
- removing spaces from pages
- centering and fixing tables spacing
- fixing modals position
